airgun/session.py

```diff
@@ -170,8 +171,11 @@ def __init__(
             to bypass login
         :param str optional url: URL path to open when starting session (without protocol
             and hostname)
+        :param str optional hostname: The hostname of a target that differs from
+            settings.satellite.hostname
         """
         self.name = session_name or gen_string('alphanumeric')
+        self._hostname = hostname or settings.satellite.hostname
```

Comment:
Right now, attributes are set in the same order named parameters are defined (if you ignore public attributes between them). I think it would be nice to follow that convention.

Reply:
Sure, I can change that